### PR TITLE
Reviews feature

### PR DESCRIPTION
**Pull request by Maxisthemoose**
Epic Review features lessgo

### CODE REVIEW

File: server/api/reviews.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+    const { id } = req.params;
+    const { userID, review } = req.body;
+
+    // Check if the bot exists
+    const foundBot = await Bots.findOne({ id });
+    if (!foundBot) return res.status(404).json({ message: "That bot doesn't exist in the database.", error: "Not Found."});
+
+    // Check if the user already reviewed this bot
+    const userReviewd = await Reviews.findOne({ botId: id, userID });
```
Comment (CreeperPlanet26) [inline server/api/reviews.js:18]: It's called userId in the database so i am gonna update it because it is userId now not userID

Comment (Maxisthemoose) [reply server/api/reviews.js:18]: Yeah, I thought I had changed all of them, but I guess I didn't, thats fine

Comment (CreeperPlanet26) [reply server/api/reviews.js:18]:
```js
else return res.status(200).json({ message: "Successfully deleted the review from the database." });
``` 
Are u actually deleting the thing?

File: server/api/reviews.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    // Check if the bot exists
+    const foundBot = await Bots.findOne({ id: botId });
+    if (!foundBot) return res.status(404).json({ message: "That bot doesn't exist in the database.", error: "Not Found." });
+
+    const foundReview = await Reviews.findById({ _id: reviewId });
+    if (!foundReview) return res.status(404).json({ message: "That review doesn't exist in the database.", error: "Not Found" });
+
+    else return res.status(200).json({ message: "Successfully deleted the review from the database." });
+
+});
```
Comment (CreeperPlanet26) [inline server/api/reviews.js]: Are you actually deleting it or am I just blind?

Comment (Maxisthemoose) [reply server/api/reviews.js]: Huh?

**Comment by CreeperPlanet26** [comment]
I am about to fix the merge issues

**Comment by Maxisthemoose** [comment]
Sounds good

File: server/api/reviews.js (hunk 2 above)
Comment (CreeperPlanet26) [reply server/api/reviews.js]: like where is the .delete() or whatever

Comment (Maxisthemoose) [reply server/api/reviews.js]: Done

**Comment by CreeperPlanet26** [comment]
LETS GOOO

**Pull request merged**